compiler/test/src/test/java/org/apache/nemo/compiler/frontend/beam/BeamFrontendALSTest.java

```diff
@@ -36,27 +36,23 @@
 @RunWith(PowerMockRunner.class)
 @PrepareForTest(JobLauncher.class)
 public final class BeamFrontendALSTest {
-  @Test
+  // TODO #260: Beam Accumulator-based Partial Aggregation
+  // @Test
```

Comment:
?

Reply:
We don't have to disable it in this issue because this issue is just refactoring. No logic changed